pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -913,10 +903,24 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 			if err != nil {
```

Comment:
could you move the error check right after line [901](https://github.com/kubevirt/kubevirt/pull/2742/files#diff-caee6d384498fde52fde15891ac547aaR901)?